definitions/variable/grids infrastructure/gas_grid.yaml

```
@@ -0,0 +1,51 @@
+- Capacity|Natural Gas|Regasifier:
+        description: Regasifier upperbound capacity
+        unit: GW
+- Capacity|Hydrogen|Regasifier:
+        description: Regasifier upperbound capacity
+        unit: GW
+- Capacity|Natural Gas|Storage|Aquifer:
+        description: Aquifer storage capacity
+        unit: GWh
+- Capacity|Natural Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage capacity
+        unit: GWh
+- Capacity|Natural Gas|Storage|Depleted Field:
+        description: Depleted Field storage capacity
+        unit: GWh
+- Injection Rate|Natural Gas|Storage|Aquifer:
+        description: Aquifer storage Injection Rate
+        unit: GW
+- Injection Rate|Natural Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage Injection Rate
+        unit: GW
+- Injection Rate|Natural Gas|Storage|Depleted Field:
+        description: Depleted Field storage Injection Rate
+        unit: GW
+- Extraction Rate|Natural Gas|Storage|Aquifer:
+        description: Aquifer storage Extraction Rate
+        unit: GW
+- Extraction Rate|Natural Gas|Storage|Salt Cavern:
+        description: Salt Cavern storage Extraction Rate
+        unit: GW
+- Extraction Rate|Natural Gas|Storage|Depleted Field:
+        description: Depleted Field storage Extraction Rate
+        unit: GW
+- Network|Natural Gas|Capacity:
+        description: Capacity of the pipeline
+        unit: GW/h
+- Network|Hydrogen|Capacity:
+        description: Capacity of the pipeline
+        unit: GW/h
+- Network|Hydrogen|Length:
+        description: Length of the pipeline
+        unit: km
+- Network|Natural Gas|Length:
+        description: Length of the pipeline
+        unit: km
+- Network|Natural Gas|Length|Offshore:
+        description: Length of the offshore part of the pipeline
+        unit: km
+- Network|Hydrogen|Length|Offshore:
+        description: Length of the offshore part of the pipeline
+        unit: km
```

Always add an empty line at the end of a file
```suggestion
        unit: km

```